### PR TITLE
Add validates and isValidatedBy relationships. Add attribute validation

### PR DESCRIPTION
**Pull request by meleemistress**
by enum

### CODE REVIEW

File: model/src/main/java/com/chiralbehaviors/CoRE/WellKnownObject.java [hunk 1]
```diff
@@ -1291,6 +1291,32 @@ public String wkoName() {
...
+        IS_VALIDATED_BY() {
+
+            @Override
+            public String description() {
+                return "A is validated by B";
+            }
+
+            @Override
+            public WellKnownRelationship inverse() {
+                return WellKnownRelationship.VALIDATES;
```
Comment (Hellblazer) [inline model/src/main/java/com/chiralbehaviors/CoRE/WellKnownObject.java]: shouldn't this be "validates enum" or something?  Validates seems pretty broad and won't lend it self to doing different validations.

File: model/src/main/java/com/chiralbehaviors/CoRE/attribute/Attribute.java [hunk 2]
```diff
@@ -77,8 +77,8 @@
                @NamedQuery(name = FIND_BY_NAME, query = "select e from Attribute e where e.name = :name"),
                @NamedQuery(name = GET_CHILD, query = "SELECT rn.child "
                                                      + "FROM AttributeNetwork rn "
-                                                     + "WHERE rn.parent = :parent "
```
Comment (Hellblazer) [inline model/src/main/java/com/chiralbehaviors/CoRE/attribute/Attribute.java]: ?

Comment (meleemistress) [inline model/src/main/java/com/chiralbehaviors/CoRE/attribute/Attribute.java]: goddamnit

Comment (meleemistress) [inline model/src/main/java/com/chiralbehaviors/CoRE/attribute/Attribute.java]: I think... I *think* this is github being stupid. I've checked locally, and the changes are correct there, and there are no diffs between github and my repo

**Pull request merged**